openupgrade_scripts/scripts/project/15.0.1.2/post-migration.py

```diff
@@ -0,0 +1,22 @@
+from openupgradelib import openupgrade
+
+
+def _convert_field_m2o_to_m2m(env):
+    # Convert m2o to m2m in 'project.task'
+    openupgrade.m2o_to_x2m(
+        env.cr, env["project.task"], "project_task", "user_ids", "user_id"
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "project", "15.0.1.2/noupdate_changes.xml")
+    openupgrade.delete_record_translations(
+        env.cr,
+        "project",
+        [
+            "mail_template_data_project_task",
+            "rating_project_request_email_template",
```

Comment:
@hoangtiendung070797 `rule` có update `name` thì cũng cần xóa nhé
```suggestion
            "rating_project_request_email_template",
            "project_project_rule_portal",
```